Source/FieldSolver/SpectralSolver/SpectralSolver.H

```diff
@@ -26,26 +27,31 @@ class SpectralSolver
                         const amrex::RealVect dx, const amrex::Real dt,
                         const bool pml=false );
 
-        /* \brief Transform the component `i_comp` of MultiFab `mf`
+        /**
+         * \brief Transform the component `i_comp` of MultiFab `mf`
          *  to spectral space, and store the corresponding result internally
          *  (in the spectral field specified by `field_index`) */
-        void ForwardTransform( const amrex::MultiFab& mf,
+         void ForwardTransform( const amrex::MultiFab& mf,
```

Comment:
this line should not be moved :) just revert
```suggestion
        void ForwardTransform( const amrex::MultiFab& mf,
```